tools/azsdk-cli/Azure.Sdk.Tools.Cli/Models/Responses/TspClientUpdateResponse.cs

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using System.Text;
+using System.Text.Json.Serialization;
+
+namespace Azure.Sdk.Tools.Cli.Models;
+
+/// <summary>
+/// Response payload for TspClientUpdateTool MCP / CLI operations.
+/// </summary>
+public class TspClientUpdateResponse : Response
+{
+    [JsonPropertyName("session")]
+    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+    public UpdateSessionState? Session { get; set; }
+
+    [JsonPropertyName("message")]
+    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+    public string? Message { get; set; }
+
+    public override string ToString()
+    {
+        var sb = new StringBuilder();
+        if (!string.IsNullOrEmpty(Message))
+        {
+            sb.AppendLine(Message);
+        }
+        if (Session != null)
+        {
+            sb.AppendLine($"Session: {Session.SessionId} Status: {Session.Status} API changes: {Session.ApiChanges.Count} Impacted: {Session.ImpactedCustomizations.Count} Patches: {Session.ProposedPatches.Count}");
+        }
+        return ToString(sb.ToString());
```

Comment:
nit: I just added an overload that takes `StringBuilder` types: https://github.com/Azure/azure-sdk-tools/blob/main/tools/azsdk-cli/Azure.Sdk.Tools.Cli/Models/Responses/Response.cs#L18


```suggestion
        return ToString(sb);
```

Reply:
Response.ToString() is _really_ weird - it takes a value as a parameter, but just overwrites it with whatever it wants in the body.

Can we take another look at this - I'm not sure having a function called ToString() that's not the normal ToString() is good practice.

Reply:
@richardpark-msft Including additional information in the string representation of an object is actually quite common, rather than unusual. I've seen this in Java too.
Reference: https://stackoverflow.com/a/31174312